tools/apprunner-gradle-plugin/src/main/java/org/projectnessie/quarkus/gradle/StopTask.java

```diff
@@ -29,7 +29,8 @@ public StopTask() {
   public void stop() {
 
     if (application == null) {
-      getLogger().warn("No application found.");
+      getLogger().debug("No application found.");
```

Comment:
Shouldn't this be higher priority? If stop is being run w/o an application doesn't that indicate an error?

Reply:
It would always log a warning, if the tests don't run, because we cannot make the start+stop tasks cacheable, because you cannot make a task's execution depend on another task execution.